client/src/app/components/modify-client-role/modify-client-role.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardService } from 'src/app/services/dashboard.service';
+
+@Component({
+  selector: 'app-modify-client-role',
+  templateUrl: './modify-client-role.component.html',
+  styleUrls: ['./modify-client-role.component.css']
+})
+export class ModifyClientRoleComponent implements OnInit {
+
+  display: boolean;
+  querrysett;
+  users: Array<CustomUser>;
+
+  constructor(private dashboardService: DashboardService) {
+    this.display = false;
+    this.users = new Array<CustomUser>(0);
+  }
+
+  ngOnInit(): void {
+  }
+
+  obtainClients(): void
+  {
+    this.dashboardService.getAllClients().subscribe(val =>
+      {
+        this.querrysett = JSON.parse(JSON.parse(val));
+        if (this.users.length !== 0)
+        {
+          this.users = new Array<CustomUser>(0);
+        }
+        for (const value of this.querrysett)
+        {
+          this.users.push(new CustomUser(value.fields));
+        }
+      });
+  }
+
+}
+
+class CustomUser
```

Comment:
We had discussed the utility of this class. I think it can be removed with some minor tweaks to the obtainClients() method. I'll look into it.

Reply:
I moved this class to the models directory. I then refactored the request to return an array of Users directly. This cut done alot of the code from inside the obtainClients() method right above this. 1303838439ed5deedd2c6edad9442ee3883d0edf
Also I made it so only relevant fields are returned from the request (no primary key or password for example) 5cceee12eb884b2d49901094c825eed86230e309